multipaz-doctypes/src/commonMain/kotlin/org/multipaz/documenttype/knowntypes/AgeVerification.kt

```diff
@@ -21,40 +20,18 @@ object AgeVerification {
         with(DocumentType.Builder("Age Verification Credential")) {
             addMdocDocumentType(AV_DOCTYPE)
 
-            // Attributes.
-            addMdocAttribute(
-                type = DocumentAttributeType.Boolean,
-                identifier = "age_over_18",
-                displayName = "Older Than 18",
-                description = "Age over 18?",
-                mandatory = true,
-                mdocNamespace = AV_NAMESPACE,
-                icon = Icon.TODAY,
-                sampleValue = SampleData.AGE_OVER_18.toDataItem()
-            )
-            addMdocAttribute(
-                type = DocumentAttributeType.Picture,
-                identifier = "portrait",
-                displayName = "Photo of Holder",
-                description = "A reproduction of the Age Verification Credential holder’s portrait.",
-                mandatory = false,
-                mdocNamespace = AV_NAMESPACE,
-                icon = Icon.ACCOUNT_BOX,
-                sampleValue = SampleData.PORTRAIT_BASE64URL.fromBase64Url().toDataItem()
-            )
-            val actualAge = SampleData.AGE_IN_YEARS
-            val additionalAgeThresholds = listOf(13, 15, 16, 21, 23, 25, 27, 28, 40, 60, 65, 67)
-            for (age in additionalAgeThresholds) {
-                val isOverNN = actualAge > age
+            // age_over_NN.
+            val sampledAges = listOf(13, 15, 16, 18, 21, 23, 25, 27, 28, 40, 60, 65, 67)
+            for (age in IntRange(0, 99)) {
```

Comment:
This should probably be IntRange(1, 99) since `age_over_00` doesn't seem to make sense (always true) ...